docs/docs/domain.mdx

```diff
@@ -809,13 +809,15 @@ action_endpoint:
 ```
 **After selective domain for custom actions is enabled, domain will be sent only to
 those custom actions which have specifically stated that they need it.**
+`validate_form_action` is an exception to this rule as it will always have the domain sent to it.
```

Comment:
This is gonna be true for any action which starts with `validate_`.
@ancalita 
Do you maybe know it we have other actions which start with `validate_`?

Reply:
@Tawakalt That is correct, I think we need to remove the code part (e.g. `validate_form_action`) and instead say something more generic like `custom actions inheriting from rasa-sdk FormValidationAction parent class (link to the part of documentations for this)`

Reply:
That makes perfect sense @ancalita